src/app/(sidebar)/explorer/components/Time.tsx

```diff
@@ -0,0 +1,16 @@
+function formatDate(date: Date): string {
+  const formatter = new Intl.DateTimeFormat("en-US", {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+    hour12: true,
+    hour: "numeric",
+    minute: "numeric",
+  });
+  return formatter.format(date);
+}
+
+export function Time({ timestamp }: { timestamp: number }) {
+  const date = new Date(timestamp * 1000);
+  return <time title={date.toISOString()}>{formatDate(date)}</time>;
+}
```

Comment:
Could we use `formatTimestamp()` helper instead of this component?

Reply:
I'd like to keep this just because I want to display the full timestamp as well. But I used the existing `formatTimestamp` function.